llvm/include/llvm/Analysis/LoopAccessAnalysis.h

```diff
@@ -366,16 +366,20 @@ class MemoryDepChecker {
 
   struct DepDistanceStrideAndSizeInfo {
     const SCEV *Dist;
-    uint64_t StrideA;
-    uint64_t StrideB;
+    uint64_t MaxStride;
```

Comment:
While you are here, it might be worth documenting which unit is used for stride here (i.e what unscaled means here)

Reply:
Fixed, thanks.